### PR TITLE
rootston: fix crash with fullscreen surfaces

### PR DESCRIPTION
**Pull request by myfreeweb**
Fixes RetroArch fullscreen mode.

### CODE REVIEW

**Comment by emersion** [comment]
Hmm. Maybe we could un-fullscreen when un-mapping views instead?

**Comment by emersion** [comment]
Okay, I see that this also happens before views are mapped. Can you move the check at the beginning of `view_for_each_surface`? With ioquake3 this segfault happens in the presentation feedback codepath.

**Comment by myfreeweb** [comment]
Updated. Does this fix ioq3?

File: rootston/output.c [hunk 1]
```diff
@@ -67,6 +67,9 @@ static void surface_for_each_surface(struct wlr_surface *surface,
 static void view_for_each_surface(struct roots_view *view,
 		struct layout_data *layout_data, wlr_surface_iterator_func_t iterator,
 		void *user_data) {
+	if (!view->wlr_surface)
```
Comment (emersion) [inline rootston/output.c]: Style: please always use brackets

**Comment by emersion** [comment]
Yes, this does fix ioquake3 :)

**Comment by emersion** [comment]
Thanks!

**Pull request merged**